prow/report/report.go

```diff
@@ -233,10 +235,14 @@ func createComment(reportTemplate *template.Template, pj kube.ProwJob, entries [
 		"--- | --- | --- | ---",
 	}
 	lines = append(lines, entries...)
+	if reportTemplate != nil {
+		lines = append(lines, []string{
+			"",
+			b.String(),
+			"",
```

Comment:
I think this empty string needs to be moved to the next append statement to properly add an empty line if after the table if there is no report template.